src/main/java/com/cgdp/library/validation/ValidationUtility.java

```diff
@@ -0,0 +1,53 @@
+package com.cgdp.library.validation;
+
+import java.time.LocalDate;
+import java.util.function.Supplier;
+
+public class ValidationUtility {
+
+    public static Long required(String paramName, Long value) {
+        validate(() -> value == null, "%s must not be null", paramName);
+        return value;
+    }
+
+    public static String requiredNotBlank(String paramName, String value) {
+
+        validate(() -> value == null, "%s must not be blank", paramName);
+        return value;
+    }
+
+    public static LocalDate validatePublicationYear(String paramName, LocalDate value) {
```

Comment:
Why such specific name? We should make things reusable when possible, change to more accurate name